pkg/models/tasks.go

```diff
@@ -1138,6 +1139,14 @@ func (t *Task) Update(s *xorm.Session, a web.Auth) (err error) {
 			return
 		}
 	}
+	// When a task is marked as done, increment the counter
+	if !ot.Done && t.Done {
```

Comment:
Please use an event for this instead of a goroutine.

https://vikunja.io/docs/events-and-listeners/